docs/plugins/repository-s3.asciidoc

```diff
@@ -34,10 +34,7 @@ PUT _snapshot/my_s3_repository
 [[repository-s3-client]]
 ==== Client Settings
 
-The client used to connect to S3 has a number of settings available. Client setting names are of
-the form `s3.client.CLIENT_NAME.SETTING_NAME` and specified inside `elasticsearch.yml`.  The
-default client name looked up by a `s3` repository is called `default`, but can be customized
-with the repository setting `client`. For example:
+The client used to connect to S3 has a number of settings available. Settings are either "public" or "secure", Public client settings may  be specified in the `elasticsearch.yml` file and are of the form `s3.client.CLIENT_NAME.SETTING_NAME`. The default client name looked up by a `s3` repository is called `default`, but can be customized with the repository setting `client`. For example:
```

Comment:
Rather than introducing the idea of "public" settings, I'd suggest something like this:
"The client ... has a number of settings available. All of these settings can be added to the elasticsearch.yml configuration file, with the exception of the secure settings, which you add to the Elasticsearch keystore. The settings have the form..."